### PR TITLE
destroy action

### PR DESCRIPTION
**Pull request by tildeee**

### CODE REVIEW

File: intro-to-rails/destroy-action.md [hunk 1]
```diff
@@ -1 +1,121 @@
...
+- Use a route parameter in a controller action
+- Learn to make a `delete` request
+
+## The `destroy` Action
+
+The `destroy` action is the last of the RESTful controller actions we will define. Its responsibility is, given some information (through route params), find a specific item, and then delete it from the database. **It is the _Delete_ in CRUD.** The `destroy` action is a _Rails convention_, so we will continue to use the name `destroy`.
+
+## Predict with Testing
+
+What should our `destroy` action do?  In our controller tests, we should verify that the record is deleted, and that the number of records decreases by one.
```
Comment (CheezItMan) [inline intro-to-rails/destroy-action.md:18]: Love the test 1st approach.

File: intro-to-rails/destroy-action.md [hunk 2]
```diff
@@ -1 +1,121 @@
...
+1. Tests: We will write two controller tests-- one for nominal case, and one for edge case.
+
+### View: Adding a Link to the `destroy` Action
+
+In our active site, we will need some way for a user to delete a book.  We can do so with a link, but `link_to` by default makes a `GET` request.  We can add a `link_to` with a `method` attribute similar to this, in our `index.html.erb` and `show.html.erb` views.
+
+```erb
+<%= link_to "Delete #{book.title}", book_path(book.id), method: :delete %>
+```
+
```
Comment (CheezItMan) [inline intro-to-rails/destroy-action.md:67]:
```suggestion

<details style="max-width: 700px; margin: auto;">
  <summary>What happens if you leave off the `method: :delete` from the `link_to</summary>
  
    the link will instead go to the item's show page
  </details>
```

File: intro-to-rails/destroy-action.md [hunk 3]
```diff
@@ -1 +1,121 @@
...
+## Exercise: Tests! Write the Destroy Action Tests
+
+Now that we've seen the implementation for a working controller action, let's reverse-engineer how we would test it.
+
+Follow these directions:
+
+1. First, read through the implemented controller action. Make a list of every line of code that you could test.
+2. Go through your list of possible things to test. Revise your list so you only have a few things to test that "prove the controller action is working."
+3. Then, write your first draft of two tests for the destroy action. One test should be the "nominal case" and one should be an "edge case," that anticipates something atypical happening. Feel free to use the code samples above.
+4. Lastly, compare your tests to a solution for the controller tests [here](https://raw.githubusercontent.com/Ada-Developers-Academy/textbook-curriculum/master/08-rails/code_samples/destroy_controller_tests.rb).
```
Comment (CheezItMan) [inline intro-to-rails/destroy-action.md:112]: To make Devin happy

```suggestion
1. First, read through the implemented controller action. Make a list of every line of code that you could test.
1. Go through your list of possible things to test. Revise your list so you only have a few things to test that "prove the controller action is working."
1. Then, write your first draft of two tests for the destroy action. One test should be the "nominal case" and one should be an "edge case," that anticipates something atypical happening. Feel free to use the code samples above.
1. Lastly, compare your tests to a solution for the controller tests [here](https://raw.githubusercontent.com/Ada-Developers-Academy/textbook-curriculum/master/08-rails/code_samples/destroy_controller_tests.rb).
```

**Review by CheezItMan — APPROVED** [review]
Nice, this works.  I left a few suggestions, but this will work.
